zstacklib/zstacklib/utils/sanlock.py

```
@@ -441,18 +500,57 @@ def get_lockspace(vg_uuid):
+@lock.lock("sanlock_backup")
+def backup_lockspace_metadata(vg_uuid, host_id):
+    bk_file = os.path.join(LOCKSPACE_BACKUP_PATH, "%s.lockspace" % vg_uuid)
+    r, o, e = read_delta_lease(vg_uuid, host_id)
+    if r != 0:
+        return
+    context = {}
+    meta_backup = linux.read_file(bk_file)
+    if meta_backup:
+        try:
+            context = jsonobject.loads(meta_backup).__dict__
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)